### ISSUE TITLE
user story: as a user I want to see on a map how far are these places from each other

### ISSUE THREAD

**Issue by SilkeLiwan** [FACN1/nazareth-events#16]

**Comment by jbesraa** [comment]
Hi Silke,
thank you so much for raising issues and being active in here.

would it be more useful to have the distance from where the user is rather than the distance between the different place ?

**Comment by SilkeLiwan** [comment]
Yes, you are right. In a map with a scale one can figure the distances out.

**Comment by SilkeLiwan** [comment]
I closed this by mistake ....
